dags/openshift_nightlies/config/benchmarks/hcp-small-control-plane.json

```diff
@@ -0,0 +1,38 @@
+{
+    "benchmarks": [
+        {
+            "name": "node-density",
+            "workload": "kube-burner-ocp-wrapper",
+            "trigger_rule": "all_done",
+            "command": "./run.sh",
+            "env": {
+                "WORKLOAD": "node-density",
+                "LOG_LEVEL": "debug",
+                "CHURN": "false"
```

Comment:
Not actually an issue, but...
CHURN is only available on cluster-density workloads at the moment